### PR TITLE
Fix issue #674, #1221, #270

### PR DESCRIPTION
**Pull request by TianoMS**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@TianoMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (tiano). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by TianoMS** [comment]
On demand job: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/755/

File: src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Extensions/JTokenExtensions.cs [hunk 1]
```diff
@@ -73,7 +73,7 @@ internal static PSObject ToPsObject(this JToken jtoken, string objectType = null
             foreach (var property in jobject.Properties())
             {
                 psObject.Properties.Add(new PSNoteProperty(
-                    name: JTokenExtensions.ConvertToPascalCase(propertyName: property.Name), 
+                    name: property.Name,
```
Comment (vivsriaus) [inline src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Extensions/JTokenExtensions.cs]: Can we please add a test for this? Feel free to edit one of our existing resources tests to add a check for this.

Comment (TianoMS) [inline src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Extensions/JTokenExtensions.cs]: Will fix this.

**Comment by TianoMS** [comment]
On demand job: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/761/

**Comment by hovsepm** [comment]
LGTM

**Pull request merged**